lib/Backend/IntBounds.cpp

```diff
@@ -178,7 +178,7 @@ void IntBounds::SetBound(
 
     // Aggressively merge the constant lower or upper bound of the base value, adjusted by the offset
     ValueInfo const * const baseValueInfo = baseValue->GetValueInfo();
-    int constantBoundBase;
+    int constantBoundBase = 0xCCCCCCCC;
```

Comment:
>0xCCCCCCCC [](start = 28, length = 10)

Why these value?

Reply:
I couldn't find a default that made sense for these.  0 is often a bad choice, MIN/MAX_INT were not great.  

0xCCCCCCCC has a few good properties:
o Negative
o Bad pointer
o INT 3 on x86/x64
o Easy to spot in the debugger
o That's what VC uses to initialize the stack with -RTC1